src/components/dashboard/stats-table.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from "react";
+import {  IStudent } from "../../types";
+import { IUsageStats, ISupportsDef } from "../../utils/get-usage-stats";
+
+import * as css from "./stats-table.scss";
+
+interface IProps {
+  students: IStudent[];
+  stats: IUsageStats;
+}
+interface IState {
+  columnExpanded: {};
+}
+
+const Definitions = ({ definitions }: {definitions: string[]}) => (
+  <div>
+    {definitions.map((d, idx) => <p key={idx}>{d}</p>)}
+  </div>
+);
+
+const Supports = ({ supports }: {supports: ISupportsDef}) => (
+  <div>
+    <p>Text to speech: {supports.textToSpeech && "✓"}</p>
+    {supports.imageShown !== undefined && <p>Image: {supports.imageShown && "✓"}</p>}
+    {supports.videoShown !== undefined && <p>Video: {supports.videoShown && "✓"}</p>}
+  </div>
+);
+
+export default class StatsTable extends React.Component<IProps, IState> {
+  public state: IState = {
+    columnExpanded: {}
+  };
+
+  public render() {
+    const { students, stats } = this.props;
+    const terms = Object.keys(stats[students[0].id]);
```

Comment:
Right, thanks. I thought I was checking somewhere before if we pass at least one student, but won't hurt to add another check. I just made a related change in the new branch that adds proper UI to avoid touching this branch and causing merge conflicts later (also, this piece of code doesn't exist there anymore).